### ISSUE TITLE
Documentation for developers

### ISSUE THREAD

**Issue by qcaas-nhs-sjt** [lsc-sde/lsc-sde#9]
Addition of documentation for developers to get started with the various parts of the system:

* Helm
* Flux
* Terraform
* Docker Images

**Comment by qcaas-nhs-sjt** [comment]
Hi @vvcb @m1p1h 

Further to our discussion yesterday, I've added some basic guides to getting started as a developer. There will likely be other tools that are needed. For now it's primarily linking to other guides on how to install individual components.

I've loaded this into:
https://github.com/lsc-sde/lsc-sde/blob/main/Developers.md

I've also put some basic instructions on how to update our helm repositories:
https://github.com/lsc-sde/lsc-sde/tree/main/iac/helm

and instructions on each of the flux repositories telling you how to install locally onto flux. I will expand on these as well to say how to access the services locally.

I've not documented terraform or docker images yet, however I figured that the docker side of things might want to be done by one of you guys as I don't fully understand the docker images yet
